<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>991</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Search">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>701</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="searchStudentIDText">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>261</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="searchStudentID">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>StudentID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="studentSearchButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>30</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="allStudentSearchButton">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>30</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>All Search</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>701</width>
        <height>321</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>studentID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>major</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>courseName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>grade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>GPA</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Student Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="regstration">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>931</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="studentIDText">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="studentID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>StudentID</string>
       </property>
      </widget>
      <widget class="QLabel" name="name">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameText">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="major">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Major</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="majorText">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="year">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="yearText">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="registrationpushButton">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>30</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Registration</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>611</width>
        <height>351</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>studentID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>major</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>courseName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>grade</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Student Registration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete Student</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>931</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="deleteStudentText">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>201</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="deleteStudent">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>StudentID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteStudentPushButton">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>30</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Course">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add course</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="AddCourseStudentIDText">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="AddCourseStudentID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>StudentID</string>
       </property>
      </widget>
      <widget class="QLabel" name="AddCourse">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Course</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="AddCourseText">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="AddCourseGrade">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Grade</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="AddCourseGradeText">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="AddCoursePushButton">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>30</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>611</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>studentID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>major</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>courseName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>grade</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>course Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Update">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update grade</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>801</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLineEdit" name="updateStudentIDText">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="updateStudentID">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>91</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>StudentID</string>
       </property>
      </widget>
      <widget class="QLabel" name="updateCourse">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Course</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="updateCourseText">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="updateGrade">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Grade</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="updateGradeText">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>113</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="updateButton">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>30</y>
         <width>93</width>
         <height>26</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>611</width>
        <height>301</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>studentID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>major</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>year</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>courseName</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>grade</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
